src/clr-core/badge/badge.stories.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import '@clr/core/badge';
+import { cssGroup, propertiesGroup, setStyles } from '@clr/core/common';
+import { color as colorKnob, number, select } from '@storybook/addon-knobs';
+import { html } from 'lit-html';
+
+export default {
+  title: 'Components|Badge',
+  component: 'cwc-badge',
+  parameters: {
+    options: { showPanel: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/v2mkhzKQdhECXOx8BElgdA/Clarity-UI-Library---light-2.2.0?node-id=51%3A673',
+    },
+  },
+};
+
+export const API = () => {
+  const slot = number('slot', 20, undefined, propertiesGroup);
+  const badgeStatus = select(
+    'status',
+    { 'none (default gray)': undefined, info: 'info', success: 'success', warning: 'warning', danger: 'danger' },
+    undefined,
+    propertiesGroup
+  );
+  const badgeColor = select(
+    'color',
+    {
+      'gray (default)': undefined,
+      gray: 'gray',
+      purple: 'purple',
+      blue: 'blue',
+      orange: 'orange',
+      'light-blue': 'light-blue',
+    },
+    undefined,
+    propertiesGroup
+  );
+  const textColor = colorKnob('--color', undefined, cssGroup);
+  const background = colorKnob('--background', undefined, cssGroup);
+
+  return html`
+    <style>
+      cwc-badge {
+        ${setStyles({
+          '--color': textColor,
+          '--background': background,
+        })}
+    </style>
```

Comment:
The `setStyles` makes it easy to hook the storybook knobs into dynamic styles that can change the CSS properties in the storybook UI